validation_server/validation_server.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios");
+const { B3Propagator } = require("@opentelemetry/core");
```

Comment:
you require core twice. B3 is outside of the scope currently, I would remove it for now.